src/main/java/org/folio/service/file/storage/LocalFileSystemStorage.java

```diff
@@ -128,6 +128,9 @@ private String getFilePath(FileDefinition fileDefinition) {
 
   @Override
   public boolean isFileExist(String path) {
+    if (path == null) {
```

Comment:
Path empty string checking maybe useful here.

Reply:
Fixed, thank you.